internal/http/services/owncloud/ocdav/propfind/propfind.go

```diff
@@ -1714,6 +1729,15 @@ func mdToPropResponse(ctx context.Context, pf *XML, md *provider.ResourceInfo, p
 	return &response, nil
 }
 
+func hasPreview(md *provider.ResourceInfo, appendToOK func(p ...prop.PropertyXML)) {
+	_, match := thumbnail.SupportedMimeTypes[md.MimeType]
```

Comment:
I guess this will bite us as soon as we implement a configuration option in the thumbnails service. Especially when thumbnails and webdav are running separately. Would be cool if we had some type of runtime discovery fo that.

Reply:
@rhafer see my second commit with config override. I think we should keep it as KISS as possible. Maybe this is too much.